docs/GUIDE.en.md

```diff
@@ -1249,6 +1250,132 @@ near account \
+
+```txt
+Public key (printed to stdout): 
+ed25519:FsRjjvkQZbwcBooXyuz4WMxXtxEKLJVJ6nc3CnaurdRr
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/jlZk2P1wCLJG6tpR8YeNONNBr?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/jlZk2P1wCLJG6tpR8YeNONNBr.png" width="836"/>
+</a>
+</details>
+
+#### from-seed-phrase - Get the public key with the seed phrase
+
+To get the public key from the seed phrase, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-seed-phrase 'trigger arrow grunt vendor crane safe reflect please sponsor verify club shiver' \
+        --seed-phrase-hd-path 'm/44'\''/397'\''/0'\'''
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
+
+```txt
+Public key (printed to stdout): 
+ed25519:3fm1ctizEANiJG2CgJXx41e18BjtNTAnB4hfYSMjd4Fh
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/85kbNU8gg0i0LU1Cm9n9umM78?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/85kbNU8gg0i0LU1Cm9n9umM78.png" width="836"/>
+</a>
+</details>
+
+#### from-plaintext-private-key - Get the public key from the plaintext private key
+
+To get the public key from the plaintext private key, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-plaintext-private-key ed25519:3AoMxLat91aAdkh4vyq7MgbKepYhSiC5WzknLFbiXUKfsoCXXeuN9W6R4EpFd3TLvBms7gbafupvtvQJmBt7W24f
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
+
+```txt
+Public key (printed to stdout): 
+ed25519:3fm1ctizEANiJG2CgJXx41e18BjtNTAnB4hfYSMjd4Fh
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/XV4rJx6VVxQaKDVoDz2l6Vudt?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/XV4rJx6VVxQaKDVoDz2l6Vudt.png" width="836"/>
+</a>
+</details>
+
+#### from-keychain - Get the public key stored in a secure keychain
+
+To get the public key from a secure keychain, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-keychain volodymyr.testnet \
+    network-config testnet
+```
+
+<details><summary><i>The result of this command will be as follows:</i></summary>
+
+```txt
+Public key (printed to stdout): 
+ed25519:CuXJ7CSpDdHvoKMwA4whFnaUwQuWGNCLu7APHKiQb4az
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/9Ct7taSDiQa6BsW5z7iZtXXlA?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/9Ct7taSDiQa6BsW5z7iZtXXlA.png" width="836"/>
+</a>
+</details>
+
+#### from-legacy-keychain - Get the public key stored in the legacy keychain (compatible with the old near CLI)
```

Comment:
nit: this could provide a link to disambiguate what `old near CLI` means

Reply:
nit: i had most problems with this command until changing error msg on miss about which file specifically was queried. 
Without full implementation details, doc can state that key is searched for in a file, whose path is computed from account-name + network + maybe smth else , according to some pattern.

Reply:
Done